src/screens/OtherProfile/index.js

```diff
@@ -195,11 +193,7 @@ const OtherProfile = () => {
   };
 
   const handleSetFollow = async () => {
-    setDataMain((prevState) => ({
-      ...prevState,
-      is_following: true,
-      is_me_following_target: true
-    }));
+    setIsCurrentFollowed(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 182, this change simplifies the state management by only updating the `isCurrentFollowed` state. Again, please ensure that there are no other parts of your application that rely on the `is_following` or `is_me_following_target` properties of `dataMain`, as these are no longer being updated here.


<!-- This is an auto-generated comment by OSS CodeRabbit -->